### DISCUSSION TITLE
Sun disc appearing from IndirectLight?

### DISCUSSION THREAD

**Discussion by ss9326** [google/filament#7371] [Q&A] [answered]
I'm working on an ARCore application with a Sceneform clone using Filament 1.21. When I use KTXLoader to load an environment texture (KTX file generated using cmgen from HDR) passed as indirectLight (no Skybox), I get a Sun disc appearing on my reflective material.

This Sun doesn't seem to be a controllable entity on the Scene like regular Light instances, so I can't find a way to toggle it on/off or change its intensity etc.

I'm new to filament so I'd really appreciate someone shedding some light (no pun) on how this Sun disc is rendered, and if I can remove/modify it.

**Comment by romainguy** [comment, marked as answer]
No sun light is created on your behalf. Either it's part of the environment map or Sceneform is adding it for you.
